daemon/cmd/daemon.go

```diff
@@ -987,25 +1008,28 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	// iptables rules can be updated only after d.init() intializes the iptables above.
 	err = d.updateDNSDatapathRules()
 	if err != nil {
+		log.WithError(err).Error("error encountered while updating DNS datapath rules.")
 		return nil, restoredEndpoints, err
```

Comment:
Is there a reason not to wrap these errors? (Is it because we're concerned about concatenating a long series of errors together..?)

Reply:
I think line 1011 is a miss. 

But line 942 - d.allocateIPs logs and fatals inside that function I believe.

Reply:
Okay fixed.